### DISCUSSION TITLE
Welcome to ChatGPT.Net Discussions!

### DISCUSSION THREAD

**Discussion by PawanOsman** [PawanOsman/ChatGPT.Net#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jsrc1990** [comment]
Hi, Excuse me, where do i get the SessionToken? It should not be API Key?

**Comment by PawanOsman** [comment]
> Hi, Excuse me, where do i get the SessionToken? It should not be API Key?

Hey, you can get your account session token from your browser cookie
First login to chatgpt then check your browser cookies

**Reply by IvanKonov** [reply]
![image](https://user-images.githubusercontent.com/20556427/213882418-991a094b-6e9b-49eb-8693-752aa724d385.png)

**Reply by PanamaGit** [reply]
> ![image](https://user-images.githubusercontent.com/20556427/213882418-991a094b-6e9b-49eb-8693-752aa724d385.png)

Hi, what could be the problem?

code from cmd:

```
Connected to the Bypass Node!
Введите текст
привет, как дела?
Authentication failed. Please ensure that your SessionToken is valid. If you have 5 failed authentications, your IP will be suspended for one minute. you have 1 failed authentications.
Incorrect API key provided: null. You can find your API key at https://beta.openai.com.
```

**Reply by PawanOsman** [reply]
> Hi, what could be the problem?
> 
> code from cmd:
> 
> ```
> Connected to the Bypass Node!
> Введите текст
> привет, как дела?
> Authentication failed. Please ensure that your SessionToken is valid. If you have 5 failed authentications, your IP will be suspended for one minute. you have 1 failed authentications.
> Incorrect API key provided: null. You can find your API key at https://beta.openai.com.
> ```

delete the JSON configuration file and renew your session token

**Comment by vinayakj-futurism** [comment]
Hi,
I found difference between the answers provided by real time open AI response and from this tool, for the same question. May I know what could be the reason for this difference?

Thanks,
